src/shogun/features/DenseFeatures.cpp

```diff
@@ -352,6 +355,12 @@ template<class ST> void CDenseFeatures<ST>::copy_feature_matrix(SGMatrix<ST> src
 	initialize_cache();
 }
 
+template<class ST> Eigen::Map < Eigen::Matrix< ST, -1, -1, 0,-1,-1>, 0, Eigen::Stride<0,0>> CDenseFeatures<ST>::get_eigen_map()
```

Comment:
you could define a `typedef` for this, like `CDenseFeaturesEigenMap`:
```
typdef Eigen::Map < Eigen::Matrix< ST, -1, -1, 0,-1,-1>, 0, Eigen::Stride<0,0>> CDenseFeaturesEigenMap<ST>
```
and shoudn't we return a reference?